pom.xml

```diff
@@ -36,6 +36,30 @@
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
 		</dependency>
 		
+		<dependency>
+			<groupId>de.flapdoodle.embed</groupId>
+			<artifactId>de.flapdoodle.embed.mongo</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Why are you adding this dependency?

Reply:
because I believed that this dependency was necessary for the junits, but this is not necessary

Reply:
Yes, I agree, please delete this dependency if you're not using it.